### ISSUE TITLE
Documentation on kaleido/chrome for server

### ISSUE THREAD

**Issue by builder173** [plotly/plotly.py#4952]
I'm trying out the release candidate builds for plotly.py and kaleido. I saw the note about chrome (or?) chromium needing to be available to kaleido now since it isn't built with that dependency. I didn't see guidance on how to accomplish that in a robust way for a python server environment. (I'm creating static images for pdfs)

**What I tried:**
The heroku chrome build-pack per this [page](https://github.com/heroku/heroku-buildpack-chrome-for-testing)
remote:        Installed Chrome dependencies for heroku-24
remote:        Adding executables to PATH
remote:        /tmp/build_f25b7b65/.chrome-for-testing/chrome-linux64/chrome
remote:        /tmp/build_f25b7b65/.chrome-for-testing/chromedriver-linux64/chromedriver
remote:        Installed Chrome for Testing STABLE version 131.0.6778.204

**What happened:**
Still getting runtime error and don't think I can reliably use the BROWSER_PATH to point plotly to this installation.
RuntimeError: Kaleido now requires that chrome/chromium is installed separately. Kaleido will try to detect it automatically, but the environmental error "BROWSER_PATH" can also be set

**Comment by builder173** [comment]
TL;DR this now works out of the box and I think existing documentation is sufficient.

I came back around to testing this with following setup on heroku and my static image exports continued to work.
plotly==6.3.0
kaleido==1.0.0
choreographer==1.0.9
heroku-buildpack-chrome-for-testing

Notes: 

1. The net dyno size increase of adding the Buildpack and upgrading from legacy Kaleido was about 136Mb
2. chrome and chromedriver are on the PATH and resolve via which command in one-off dyno bash session
```
which chrome
/app/.chrome-for-testing/chrome-linux64/chrome

which chromedriver  
/app/.chrome-for-testing/chromedriver-linux64/chromedriver
```
